wgpu-types/src/lib.rs

```diff
@@ -98,7 +98,7 @@ pub const QUERY_SIZE: u32 = 8;
 /// Backends supported by wgpu.
 #[repr(u8)]
 #[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
-#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
```

Comment:
Quite a few unintentional changes here. I think they probably should be left as is due to the conditional compilation.